spec/spec_helper.rb

```diff
@@ -88,6 +90,10 @@
       config.anthropic_api_key = ENV.fetch('ANTHROPIC_API_KEY', 'test')
       config.gemini_api_key = ENV.fetch('GEMINI_API_KEY', 'test')
       config.deepseek_api_key = ENV.fetch('DEEPSEEK_API_KEY', 'test')
+      config.bedrock_api_key = ENV.fetch('AWS_ACCESS_KEY_ID', 'test')
+      config.bedrock_secret_key = ENV.fetch('AWS_SECRET_ACCESS_KEY', 'test')
+      config.bedrock_region = ENV.fetch('AWS_REGION', 'us-west-2')
```

Comment:
`test`

Reply:
This one will make refreshing the VCR cassettes pretty tricky as the region is part of the URL, host header, and authorization header. I can do it, but wanted to make sure you were aware.

Reply:
ahh! then here is fine to do that!